proposal.md

```diff
@@ -0,0 +1,7 @@
+# Proposal 
+
+## Section 1: Motivation and Purpose
```

Comment:
I did not put "Section 2" and "Section 3" for the other sections, just the section names. Do you want to remove it to be consistent? I don't think it's necessary. Or, we could add the numbering to the other sections if you feel strongly about it.

Reply:
I can remove it